proto/bpfd.proto

```diff
@@ -28,11 +29,22 @@ message LoadRequest {
     bool from_image = 2;
     string section_name = 3;
     ProgramType program_type = 4;
+    oneof attach_type {
+        NetworkMultiAttach network_multi_attach = 5;
```

Comment:
Can you explain your thoughts behind the multi and single attach terminology?  It seems like we can have multiple programs attached in both cases.  It's just in the NetworkMultiAttach case (e.g., XDP or TC) we are using a dispatcher.

Reply:
I was trying to differentiate between:

1. BPF programs where the kernel takes care of ordering, and we don't care
2. BPF programs where we take care of the ordering

I'm not 100% sold on the name though.